onnxruntime/core/optimizer/initializer.h

```diff
@@ -584,6 +668,18 @@ class Initializer final {
         }
         break;
       }
+      case ONNX_NAMESPACE::TensorProto_DataType_BFLOAT16: {
+        uint16_t* dst = data<uint16_t>();
+        const uint16_t* src = other.data<uint16_t>();
+        for (int i = 0; i < n; i++) {
+          int index = other.size() == 1 ? 0 : i;
+          for (int64_t j = 0; j < num; j++) {
+            auto k = i * num + j;
+            dst[k] = BFloat16((reinterpret_cast<BFloat16*>(dst + k))->ToFloat() / (reinterpret_cast<const BFloat16*>(src + index))->ToFloat()).val;
```

Comment:
Should be "*" (times) here instead of "/" (divide).

Reply:
ah, yes. thanks